src/pages/Quiz/Quiz.js

```diff
@@ -8,20 +8,36 @@ import head from '../../public/img/RicksHead.png';
  */
 // eslint-disable-next-line no-unused-vars
 function Quiz(props) {
+  const allAnswers = [];
+  const { element: counterElement, increment, getCount } = QuestionCounter();
+  const onClick = () => {
+    const answeredQuestions = getCount() - 1;
+    const points = allAnswers.filter((answer) => answer.correct).length;
+    console.log(`Quiz finished after answering ${answeredQuestions} questions!`);
+    console.log(`You scored ${points} points!`);
+  };
   // @ts-ignore
-  const { question } = Question(props.generator.next().value);
+  let question = Question(props.generator.next().value);
+
   return html`<div class="${styles.wrapper}">
   ${BackgroundDecoration()}
     <div class="${styles.quizContainer}">
+      <div class="${styles.counterElementWrapper}">${counterElement}</div>
+      <div class="${styles.popupCloseWrapper}">${PopupClose({ onClick })}</div>
```

Comment:
Ten onClick zamknięcie quizu, to jak to będzie działać? Będzię kierowało w to miejsce co zakończenie quizu gdy czas się skończy?